### DISCUSSION TITLE
Possible Import/Export of profiles?

### DISCUSSION THREAD

**Discussion by themadtux** [Hekili/hekili#931] [Ideas]
Is there any possibility to add an import/export to the profile to share the layout with others etc?  I'm not a coder otherwise I'd add it to the project :(

Just an idea.. 

Love the addon thanks for making it!

**Comment by Hekili** [comment]
Not whole profiles, no -- there's really not a need for it.

You can import/export displays/layouts in /hekili > Displays > Share Styles.

**Reply by themadtux** [reply]
Ahh I hadn't noticed that.  Thanks for the update :)

Cheers!
